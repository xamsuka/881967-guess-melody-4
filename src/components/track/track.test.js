import React from 'react';
import renderer from 'react-test-renderer';
import Track from './track';

const track = `https://upload.wikimedia.org/wikipedia/commons/4/4e/BWV_543-fugue.ogg`;

test(`<Track /> render`, () => {
  const three = renderer
    .create(<Track audioSrc = {track} />).toJSON();

  expect(three).toMatchSnapshot();
});
