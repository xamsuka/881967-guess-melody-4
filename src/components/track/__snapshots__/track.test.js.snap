// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Track /> render 1`] = `
<div
  className="track"
>
  <button
    className="track__button track__button--play"
    type="button"
  />
  <div
    className="track__status"
  >
    <audio
      src="https://upload.wikimedia.org/wikipedia/commons/4/4e/BWV_543-fugue.ogg"
    />
  </div>
</div>
`;
